docs/en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -390,6 +390,22 @@ It is also often useful to filter by all rows with either empty or null for a gi
 	// ... WHERE "FirstName" == '' OR "FirstName" IS NULL
 	// Returns rows with FirstName which is either empty or null
 
+### Filtering by aggregates
+
+You can use aggregate expresssions in your filters, as well.
+
+	:::php
```

Comment:
Could we use GFM here instead? This syntax doesn't parse correctly sometimes depending on how you indent it ([reference](https://docs.silverstripe.org/en/4/changelogs/4.0.0/#upgrade-code-that-uses-uploadfield)).

Related discussion on #6655